dice_ml/explainer_interfaces/dice_genetic.py

```diff
@@ -402,33 +379,27 @@ def find_counterfactuals(self, query_instance, desired_range, desired_class, fea
             # rest of the next generation obtained from top 50% of fittest members of current generation
             s = self.population_size - s
             new_generation_2 = np.zeros((s, self.data_interface.number_of_features))
-            for _ in range(s):
+            for i in range(s):
```

Comment:
I know I had suggested `i` here,but looking at a few lines below at 394 is also using `i`, it feels a bit dangerous. You can perhaps use a different variable name here to avoid confusion.